package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "local-home-virual-environment",
+  "version": "0.0.1",
+  "description": "",
+  "private": true,
+  "license": "Apache-2.0",
+  "scripts": {
+    "compile": "tsc",
+    "test": "ava --verbose",
+    "lint": "eslint ."
+  },
+  "dependencies": {
+    "@google/local-home-sdk": "^1.0.0"
```

Comment:
This might be fine as a devDependencies, since it's only containing the typing

Reply:
Done.